Makefile

```diff
@@ -43,7 +43,14 @@ NEUTRON_IMG        ?= quay.io/openstack-k8s-operators/neutron-operator-index:lat
 NEUTRON_REPO       ?= https://github.com/openstack-k8s-operators/neutron-operator.git
 NEUTRON_BRANCH     ?= master
 NEUTRONAPI         ?= config/samples/neutron_v1beta1_neutronapi.yaml
-NEUTRONAPI_IMG    ?= ${SERVICE_REGISTRY}/${SERVICE_ORG}/openstack-neutron-server:current-tripleo
+NEUTRONAPI_IMG     ?= ${SERVICE_REGISTRY}/${SERVICE_ORG}/openstack-neutron-server:current-tripleo
+
+# Cinder
+CINDER_IMG       ?= quay.io/openstack-k8s-operators/cinder-operator-index:latest
+CINDER_REPO      ?= https://github.com/openstack-k8s-operators/cinder-operator.git
+CINDER_BRANCH    ?= master
+CINDER           ?= config/samples/cinder_v1beta1_cinder.yaml
```

Comment:
Yeah, I added the `TODO` on line 53 to track the need to allow customization of the actual Cinder service container images.  I'm going to address this next in a follow-up because there are multiple container images involved for Cinder and that breaks our current image customization pattern for this `install_yamls` tool.